### PR TITLE
fix: loading more posts on tall screens

### PR DESCRIPTION
**Pull request by omBratteng**
## Changes

Fixes the edge case of being unable to load more in feed when height of screen is too large.

It forces removal of the infinite scroll trigger while loading, so that the `inView` state gets reset.

Deployed to https://preview3.app.daily.dev/

WT-2087

### CODE REVIEW

File: packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export interface UseFeedInfiniteScrollProps {
 
 export const InfiniteScrollScreenOffset = classed(
   'div',
-  'absolute bottom-screen desktopL:bottom-screen-20 left-0 w-px h-px opacity-0 pointer-events-none',
+  'bottom-screen desktopL:bottom-screen-20 left-0 w-px h-px opacity-0 pointer-events-none',
```
Comment (kopancek) [inline packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts:12]: Was removal of the `absolute` class here intentional? If yes, why do we still need `left-0`?

**Review by kopancek — COMMENTED** [review]
Looks mostly good to me, just have one question.

File: packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts (hunk 1 above)
Comment (omBratteng) [reply packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts:12]: Yeah, it was intentional.
Probably don't need `left-0`. Will double check.

Comment (omBratteng) [reply packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts:12]:
```suggestion
  'bottom-screen desktopL:bottom-screen-20 w-px h-px opacity-0 pointer-events-none',
```

Comment (rebelchris) [reply packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts]: To add to that, without absolute `bottom-screen` also doesn't get added, so might want to clean it up.
Screenshot 2024-01-15 at 13 40 18

**Review by rebelchris — APPROVED** [review]
One minor addition to Milan's comment, non blocking for me.

File: packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts (hunk 1 above)
Comment (omBratteng) [reply packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts]: 🙈

File: packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export interface UseFeedInfiniteScrollProps {
 
 export const InfiniteScrollScreenOffset = classed(
   'div',
-  'absolute bottom-screen desktopL:bottom-screen-20 left-0 w-px h-px opacity-0 pointer-events-none',
+  'bottom-screen desktopL:bottom-screen-20 w-px h-px opacity-0 pointer-events-none',
```
Comment (omBratteng) [inline packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts:12]:
```suggestion
  'w-px h-px opacity-0 pointer-events-none',
```

**Review by capJavert — APPROVED** [review]

**Pull request merged**